static/js/popovers.js

```diff
@@ -519,88 +504,47 @@ function show_user_group_info_popover(element, group, message) {
     }
 }
 
-export function toggle_actions_popover(element, id) {
-    const $last_popover_elem = $current_actions_popover_elem;
-    hide_all();
-    if ($last_popover_elem !== undefined && $last_popover_elem.get()[0] === element) {
-        // We want it to be the case that a user can dismiss a popover
-        // by clicking on the same element that caused the popover.
-        return;
-    }
-
-    $(element).closest(".message_row").toggleClass("has_popover has_actions_popover");
-    message_lists.current.select_id(id);
-    const $elt = $(element);
-    if ($elt.data("popover") === undefined) {
-        const args = popover_menus_data.get_actions_popover_content_args(id);
-
-        const ypos = $elt.offset().top;
-        $elt.popover({
-            // Popover height with 7 items in it is ~190 px
-            placement: message_viewport.height() - ypos < 220 ? "top" : "bottom",
-            title: "",
-            content: render_actions_popover_content(args),
-            template: render_actions_popover_template(),
-            html: true,
-            trigger: "manual",
-        });
-        $elt.popover("show");
-        $current_actions_popover_elem = $elt;
-    }
-
-    if (window.innerWidth < media_breakpoints_num.xl) {
-        // This ensures that the popover doesn't overflow to the right of the window.
-        const actions_popover_left = window.innerWidth - $(".actions_popover_wrapper").outerWidth();
-        $(".actions_popover_wrapper").css("left", actions_popover_left);
-    }
-}
-
-export function render_actions_remind_popover(element, id) {
-    hide_all();
-    $(element).closest(".message_row").toggleClass("has_popover has_actions_popover");
-    message_lists.current.select_id(id);
-    const $elt = $(element);
-    if ($elt.data("popover") === undefined) {
-        const message = message_lists.current.get(id);
-        const args = {
-            message,
-        };
-        const ypos = $elt.offset().top;
-        $elt.popover({
-            // Popover height with 7 items in it is ~190 px
-            placement: message_viewport.height() - ypos < 220 ? "top" : "bottom",
-            title: "",
-            content: render_remind_me_popover_content(args),
-            html: true,
-            trigger: "manual",
-        });
-        $elt.popover("show");
-        current_flatpickr_instance = $(
-            `.remind.custom[data-message-id="${CSS.escape(message.id)}"]`,
-        ).flatpickr({
-            enableTime: true,
-            clickOpens: false,
-            defaultDate: "today",
-            minDate: "today",
-            plugins: [new ConfirmDatePlugin({})],
-        });
-        $current_actions_popover_elem = $elt;
-    }
-}
+// Rewrite this function when we enable this feature.
```

Comment:
Since we are not using it and this is most likely broken, I thought it was best to just comment out this code so that we can get it up and running when the feature is ready.

Reply:
I think it'd be better to either remove it more fully in a prep commit or just port it in the straightforward way; it's guarded by the `feature_flags.reminders_in_message_action_menu` feature flag which `should_display_reminder_option` feature flag, and I'd expect it'd be easy to just move the code with its current behavior (broken or not).

Reply:
Removed it since I would have to make some changes inside the function to avoid errors.